examples/src/charmory_examples/object_detection/xview/train_xview_model.py

```diff
@@ -0,0 +1,342 @@
+import albumentations as A
+
+import numpy as np
+import torch
+
+
+DEVICE = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+#DEVICE = 'cpu'
+BATCH_SIZE = 8
+import torch
+
+torch.set_float32_matmul_precision("high")
+#import armory.data.datasets
+import datasets
+from datasets import load_from_disk
+from datasets.filesystems import S3FileSystem
+
+LABEL_MAP = {
+    11: 1,  # Fixed-wing Aircraft
+    12: 2,  # Small Aircraft
+    13: 3,  # Cargo Plane
+    15: 4,  # Helicopter
+    17: 5,  # Passenger Vehicle
+    18: 6,  # Small Car
+    19: 7,  # Bus
+    20: 8,  # Pickup Truck
+    21: 9,  # Utility Truck
+    23: 10,  # Truck
+    24: 11,  # Cargo Truck
+    25: 12,  # Truck w/Box
+    26: 13,  # Truck Tractor
+    27: 14,  # Trailer
+    28: 15,  # Truck w/Flatbed
+    29: 16,  # Truck w/Liquid
+    32: 17,  # Crane Truck
+    33: 18,  # Railway Vehicle
+    34: 19,  # Passenger Car
+    35: 20,  # Cargo Car
+    36: 21,  # Flat Car
+    37: 22,  # Tank Car
+    38: 23,  # Locomotive
+    40: 24,  # Maritime Vessel
+    41: 25,  # Motorboat
+    42: 26,  # Sailboat
+    44: 27,  # Tugboat
+    45: 28,  # Barge
+    47: 29,  # Fishing Vessel
+    49: 30,  # Ferry
+    50: 31,  # Yacht
+    51: 32,  # Container Ship
+    52: 33,  # Oil Tanker
+    53: 34,  # Engineering Vehicle
+    54: 35,  # Tower Crane
+    55: 36,  # Container Crane
+    56: 37,  # Reach Stacker
+    57: 38,  # Straddle Carrier
+    59: 39,  # Mobile Crane
+    60: 40,  # Dump Truck
+    61: 41,  # Haul Truck
+    62: 42,  # Scraper/Tractor
+    63: 43,  # Front Loader/Bulldozer
+    64: 44,  # Excavator
+    65: 45,  # Cement Mixer
+    66: 46,  # Ground Grader
+    71: 47,  # Hut/Tent
+    72: 48,  # Shed
+    73: 49,  # Building
+    74: 50,  # Aircraft Hanger
+    75: 51,  # Unknown1
+    76: 52,  # Damaged Building
+    77: 53,  # Facility
+    79: 54,  # Construction Site
+    82: 55,  # Unknown2
+    83: 56,  # Vehicle Lot
+    84: 57,  # Helipad
+    86: 58,  # Storage Tank
+    89: 59,  # Shipping Container Lot
+    91: 60,  # Shipping Container
+    93: 61,  # Pylon
+    94: 62,  # Tower
+}
+
+def load_huggingface_dataset():
+    s3 = S3FileSystem(anon=False)
+    train_dataset = load_from_disk("s3://armory-library-data/datasets/train/", fs=s3)
```

Comment:
Was the `fs` argument needed? I got the impression from the docs that it would auto-infer the file system based on the URL's protocol.

Reply:
Good point, I need to take this out of my other files as well. I was just following the examples online but I guess this is not needed anymore.